app/controllers/agent/tasklist/ukproperty/PropertyIncomeSourcesController.scala

```diff
@@ -46,7 +44,7 @@ class PropertyIncomeSourcesController @Inject()(identify: IdentifierAction,
   def show(isEditMode: Boolean, isGlobalEdit: Boolean): Action[AnyContent] = (identify andThen journeyRefiner) async { implicit request =>
     subscriptionDetailsService.fetchProperty(request.reference) map { maybeProperty =>
       val formData: Map[String, String] = UkPropertyIncomeSourcesForm.createPropertyMapData(maybeProperty)
-      val form: Form[_] = ukPropertyIncomeSourceForm.fold(identity, identity)
+      val form: Form[_] = ukPropertyIncomeSourcesFormNoDate
       Ok(view(
         ukPropertyIncomeSourcesForm = form.bind(formData).discardingErrors,
         postAction = routes.PropertyIncomeSourcesController.submit(editMode = isEditMode, isGlobalEdit = isGlobalEdit),
```

Comment:
```suggestion
      Ok(view(
        ukPropertyIncomeSourcesForm = ukPropertyIncomeSourcesFormNoDate.bind(formData).discardingErrors,
        postAction = routes.PropertyIncomeSourcesController.submit(editMode = isEditMode, isGlobalEdit = isGlobalEdit),
```
Might be a little cleaner as it's not being used  elsewhere

Reply:
updated :)